spk/comskip/src/comskip.ini

```diff
@@ -0,0 +1,78 @@
+; See comskip.txt in the distribution zip file for many settable parameters, read manual.html, tuning.html and debugwindow.html for how to tune and debug comskip
+detect_method=43			; 1=black frame, 2=logo, 4=scene change, 8=fuzzy logic, 16=closed captions, 32=aspect ration, 64=silence, 128=cutscenes, 255=all
+validate_silence=1			; Default, set to 0 to force using this clues if selected above.
+validate_uniform=1			; Default, set to 0 to force using this clues (like pure white frames) if blackframe is selected above.
+validate_scenechange=1			; Default, set to 0 to force using this clues if selected above.
+verbose=10				; Show a lot of extra info, level 5 is also OK, set to 0 to disable
+max_brightness=60			; Frame not black if any pixels checked are greater than this (scale 0 to 255)
+test_brightness=40      		; Frame not pure black if any pixels checked are greater than this, will check average brightness (scale 0 to 255)
+max_avg_brightness=25			; Maximum average brightness for a dim frame to be considered black (scale 0 to 255) 0 means autosetting
+max_commercialbreak=600 		; Maximum length in seconds to consider a segment a commercial break
+min_commercialbreak=25			; Mimimum length in seconds to consider a segment a commercial break
+max_commercial_size=125			; Maximum time in seconds for a single commercial or multiple commercials if no breaks in between
```

Comment:
When viewing in full it seems to look perfect?
Did change the idents but on github it still looks wierd (using gedit)